package.json

```diff
@@ -90,7 +90,7 @@
     "oauth2orize": "latest",
     "optimist": "latest",
     "passport": "latest",
-    "passport-cas": "git://github.com/oaeproject/passport-cas#samlValidateLoginUrl",
+    "passport-cas": "0.1.0",
```

Comment:
Discussed w/ Nico and we'll fix CAS since it's not really feasible get reassuring unit test coverage on it, so it's a big risk to let the version run away from us without supervision.